index.js

```diff
@@ -31,11 +49,11 @@ function safelyGetProp (obj, prop) {
   }
 }
 
-function ensureProperties (obj) {
+function ensureProperties (obj, filterKeys, filterPaths) {
```

Comment:
I see the name of this function has not changed, but it doesn't seem like it is describing what it is doing to me